admin/services/check-state.service.js

```diff
@@ -19,11 +19,13 @@ module.exports.States = Object.freeze({
  * CheckStatus Codes
  */
 const checkStatusCodes = Object.freeze({
-  New: { code: 'NEW' },
-  Expired: { code: 'EXP' },
+  Collected: { code: 'COL' },
   Complete: { code: 'CMP' },
+  Expired: { code: 'EXP' },
+  New: { code: 'NEW' },
+  NotReceived: { code: 'NTR' },
   Started: { code: 'STD' },
```

Comment:
should we get rid of service? or at the very least identify code that may still be using it?

Reply:
Yes good idea.   Once we have moved over to the new world we can remove any dead code left behind.